sway-ir/src/optimize/memcpyopt.rs

```diff
@@ -649,6 +649,24 @@ fn local_copy_prop(
                             &mut dest_to_copies,
                         );
                     }
+                    Instruction {
+                        op:
+                            InstOp::FuelVm(
+                                FuelVmInstruction::WideBinaryOp { result, .. }
+                                | FuelVmInstruction::WideUnaryOp { result, .. }
+                                | FuelVmInstruction::WideModularOp { result, .. },
```

Comment:
Also need to check for `FuelVmInstruction::StateLoadQuadWord { load_val: result, .. },`